draft-irtf-cfrg-vdaf.md

```diff
@@ -3104,8 +3091,7 @@ as follows:
 
```

Comment:
The text above about helper shares should be updated to say "an XOF seed" instead of "XOF seeds".

Reply:
Done!